src/browser/provider/built-in/dedicated/chrome/cdp.ts

```diff
@@ -1,34 +1,75 @@
+import path from 'path';
+import os from 'os';
 import remoteChrome from 'chrome-remote-interface';
 import { GET_WINDOW_DIMENSIONS_INFO_SCRIPT } from '../../../utils/client-functions';
 
+interface Size {
+    width: number;
+    height: number;
+}
+
+interface Config {
+    headless: boolean;
+    mobile: boolean;
+    emulation: false;
+    userAgent?: string;
+    touch?: boolean;
+    width: number;
+    height: number;
+    scaleFactor: number;
+}
+
+interface ProviderMethods {
+    resizeLocalBrowserWindow (browserId: string, newWidth: number, newHeight: number, currentWidth: number, currentHeight: number): Promise<void>;
+}
+
+interface RuntimeInfo {
+    browserId: string;
+    cdpPort: number;
+    client: remoteChrome.ProtocolApi;
+    tab: remoteChrome.TargetInfo;
+    config: Config;
+    viewportSize: Size;
+    emulatedDevicePixelRatio: number;
+    originalDevicePixelRatio: number;
+    providerMethods: ProviderMethods;
+}
 
-async function getActiveTab (cdpPort, browserId) {
+interface TouchConfigOptions {
+    enabled: boolean;
+    configuration: 'desktop' | 'mobile';
+    maxTouchPoints: number;
+}
+
+const DOWNLOADS_DIR = path.join(os.homedir(), 'Downloads');
+
+async function getActiveTab (cdpPort: number, browserId: string): Promise<remoteChrome.TargetInfo> {
     const tabs = await remoteChrome.listTabs({ port: cdpPort });
-    const tab  = tabs.filter(t => t.type === 'page' && t.url.indexOf(browserId) > -1)[0];
 
-    return tab;
+    return tabs.filter(t => t.type === 'page' && t.url.indexOf(browserId) > -1)[0];
```

Comment:
Instead of `string.indexOf(subString) > -1` it's possible to use `string.includes(subString)`.
It is a little readable and shorter.